eng/pipelines/pr.yml

```diff
@@ -95,6 +96,7 @@ jobs:
     - Linux_arm64
     - Linux_musl_x64
     - Linux_x64
+    - Linux_gcc_x64
```

Comment:
Do we want to run tests for the gcc build?

Reply:
yes